seqr/views/utils/dataset_utils.py

```diff
@@ -241,16 +241,17 @@ def _get_unmatched_error(sample_keys):
 def _parse_tsv_row(row):
     return [s.strip().strip('"') for s in row.rstrip('\n').split('\t')]
 
-PROJECT_COL = 'project'
-RNA_OUTLIER_COLUMNS = {'geneID': 'gene_id', 'pValue': 'p_value', 'padjust': 'p_adjust', 'zScore': 'z_score',
-                       PROJECT_COL: PROJECT_COL}
 
+PROJECT_COL = 'project'
```

Comment:
These database/ outlier input changes allow much better shared behaviors across the RNA data types, so the code is refactored here to take advantage of that